app/models/form_object.rb

```diff
@@ -0,0 +1,29 @@
+class FormObject
```

Comment:
Added this so that it can be shared by both namespaces **Applikation** & **Evidence** (when it comes)